dll/win32/shell32/shlexec.cpp

```diff
@@ -2650,3 +2650,197 @@ HRESULT WINAPI ShellExecCmdLine(
 
     return HRESULT_FROM_WIN32(dwError);
 }
+
+/*************************************************************************
+ *                RealShellExecuteExA (SHELL32.266)
+ */
+EXTERN_C
+HINSTANCE WINAPI
+RealShellExecuteExA(
+    _In_opt_ HWND hwnd,
+    _In_opt_ LPCSTR lpOperation,
+    _In_opt_ LPCSTR lpFile,
+    _In_opt_ LPCSTR lpParameters,
+    _In_opt_ LPCSTR lpDirectory,
+    _In_opt_ LPSTR lpReturn,
+    _In_opt_ LPCSTR lpTitle,
+    _In_opt_ LPVOID lpReserved,
+    _In_ INT nCmdShow,
+    _Out_opt_ PHANDLE lphProcess,
+    _In_ DWORD dwFlags)
+{
+    SHELLEXECUTEINFOA ExecInfo;
+
+    TRACE("(%p, %s, %s, %s, %s, %p, %s, %p, %u, %p, %lu)\n",
+          hwnd, debugstr_a(lpOperation), debugstr_a(lpFile), debugstr_a(lpParameters),
+          debugstr_a(lpDirectory), lpReserved, debugstr_a(lpTitle),
+          lpReserved, nCmdShow, lphProcess, dwFlags);
+
+    ZeroMemory(&ExecInfo, sizeof(ExecInfo));
+    ExecInfo.cbSize = sizeof(ExecInfo);
+    ExecInfo.fMask = SEE_MASK_FLAG_NO_UI | SEE_MASK_UNKNOWN_0x1000;
```

Comment:
Now, we don't make use of this 0x1000 flag in ReactOS (in our shell), so I don't see why we should include here at all.
I could therefore ask you "how do you know this has to be set" ?

Reply:
![image](https://github.com/reactos/reactos/assets/2107452/03a8b0c8-e80b-4cb3-9e62-bececf5c0ba1)

Reply:
Well my question was in the context of ReactOS, since we don't make use of this flag anywhere else.
And you're showing me some IDA dump (which I then can question the legality of your implementation here...), and you don't even know for what purpose they set the 0x1000 flag there too...

Reply:
Deleting this flag is a wrong implementation, but I will delete this flag to pass this PR.

Reply:
You can add a comment instead that Windows appears to give that extra flag (whose value is 0x1000) but it doesn't appear to be used anywhere else.
If it turns out in the future someone finds the actual reason for this flag, the comment would be here and the flag be correctly introduced.

Reply:
> it doesn't appear to be used anywhere else.

Is it just lack of our vision? We are still unable to understand all of Windows but you are insisting this flag is not using in Windows.